python/kserve/kserve/model_server.py

```diff
@@ -207,7 +220,10 @@ def __init__(
         self._rest_server = None
         if self.enable_grpc:
             self._grpc_server = GRPCServer(
-                grpc_port, self.dataplane, self.model_repository_extension
+                grpc_port,
+                self.dataplane,
+                self.model_repository_extension,
+                kwargs=vars(args),
```

Comment:
Do you need to pass all the model server args to gRPCServer?

Reply:
We don't have to. But in future if we are introducing some more args for grpc server the function arguments might grow. So i thought we can just directly pass all the args as kwargs.